show-coordinates-in-multiple-formats/src/main/java/com/esri/arcgismaps/sample/showcoordinatesinmultipleformats/screens/MainScreen.kt

```diff
@@ -35,21 +49,60 @@ import com.esri.arcgismaps.sample.showcoordinatesinmultipleformats.components.Ma
 fun MainScreen(sampleName: String) {
     // create a ViewModel to handle MapView interactions
     val mapViewModel: MapViewModel = viewModel()
+    // create a map that has the WGS 84 coordinate system and set this into the map
+    val basemapLayer = ArcGISTiledLayer(LocalContext.current.applicationContext.getString(R.string.basemap_url))
+    val arcGISMap = ArcGISMap(Basemap(basemapLayer))
+    // the collection of graphics overlays used by the MapView
+    val graphicsOverlayCollection = rememberGraphicsOverlayCollection()
+    // graphics overlay for the MapView to draw the graphics
+    val graphicsOverlay = remember { GraphicsOverlay() }
+    // set up a graphic to indicate where the coordinates relate to, with an initial location
+    val initialPoint = Point(0.0, 0.0, SpatialReference.wgs84())
+
+    val coordinateLocation = Graphic(
```

Comment:
`coordinateLocation` might be confused with an actual location point. 

Could rename it to `coordinateLocationGraphic`
```suggestion
    val coordinateLocationGraphic = Graphic(
```